src/connection_manager.c

```diff
@@ -364,8 +365,19 @@ static qd_error_t load_server_config(qd_dispatch_t *qd, qd_server_config_t *conf
     config->conn_props           = qd_entity_opt_map(entity, "openProperties");       CHECK();
 
     if (strcmp(config->role, "inter-router") == 0) {
-        config->data_connection_count = qd_entity_opt_long(entity, "dataConnectionCount", 2); CHECK();
-        config->has_data_connectors   = true;
+        // For inter-router connections only, the dataConnectionCount defaults to "auto",
+        // which means it will be determined as a function of the number of worker threads.
+        if (config->data_connection_count == 0) {
```

Comment:
Since config was zero'ed above is this check necessary? Won't data_connection_count always be zero?

Reply:
true